src/network/tc/core.h

```diff
@@ -0,0 +1,10 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2019 VMware, Inc.
+ */
+#pragma once
+
+#include "time-util.h"
+
+int tc_init(double *ticks_in_usec);
+
+double tc_time_to_tick(double time);
```

Comment:
`time` -> `t`.

Also, the type of the argument and the return value should be changed.
```
uint32_t tc_time_to_tick(usec_t t);
```